### PR TITLE
feat(CalloutData): new experimental pattern created

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#2823 

### Description

The `CalloutData` pattern was created and added to the blocks folder.

### CODE REVIEW

File: packages/styles/scss/patterns/blocks/callout-data/_callout-data.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@mixin callout-data {
+  .#{$prefix}--callout {
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/callout-data/_callout-data.scss:9]: @RaphaelAmadeu this appears to be affecting other callout patterns. Can you update the class name to be specific to this pattern?

File: packages/react/src/patterns/blocks/CalloutData/CalloutData.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import Callout from '../../../internal/components/Callout/Callout';
+import { DDS_CALLOUT_DATA } from '../../../internal/FeatureFlags';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
```
Comment (kennylam) [inline packages/react/src/patterns/blocks/CalloutData/CalloutData.js:11]:
```suggestion
import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
import { featureFlag } from '@carbon/ibmdotcom-utilities/es/utilities/featureFlag/featureFlag';
```

**Comment by jeffchew** [comment]
@RaphaelAmadeu jest snapshots are also out of date.

**Comment by jeffchew** [comment]
@RaphaelAmadeu circleci is complaining about an import for the feature flag, can you take a look?

**Comment by jeffchew** [comment]
@RaphaelAmadeu snapshots are outdated now, can you update?

File: packages/styles/scss/patterns/blocks/callout-data/_callout-data.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+          padding-top: $layout-01;
+          color: $text-05;
+        }
+      }
+
+      &__column {
+        height: auto;
+
+        @include carbon--breakpoint('md') {
+          height: 320px;
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/callout-data/_callout-data.scss:50]: Can you use the miniunit mixin for this?

File: packages/styles/scss/patterns/blocks/callout-data/_callout-data.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      }
+
+      &__content {
+        padding-left: 0;
+        margin-left: 0;
+        display: flex;
+        height: 100%;
+        flex-direction: column;
+        width: 90%;
+        max-width: 640px;
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/callout-data/_callout-data.scss:73]: Same here, use the carbon mixin.

**Pull request merged**